app/views/content_items/manuals/_updates.html.erb

```diff
@@ -0,0 +1,52 @@
+<div class="govuk-grid-row">
+  <div class="manual-body" id="content">
+    <article aria-labelledby="section-title">
+      <div class="govuk-grid-column-full">
+        <%= render "govuk_publishing_components/components/heading", {
+          font_size: "l",
+          heading_level: 1,
+          id: "section-title",
+          margin_bottom: 4,
+          text: t("manuals.updates_title", title: content_item.title),
+        } %>
+      </div>
+
+      <% content_item.presented_change_notes.each do |year, updates_by_year| %>
+        <div class="govuk-grid-column-two-thirds">
+          <%= render "govuk_publishing_components/components/heading", {
+            text: year,
+            font_size: "l"
+          } %>
+
+          <%= render "govuk_publishing_components/components/accordion", {
+            heading_level: 3,
+            items: updates_by_year.map do |day, updated_documents|
+              accordion_content = capture do %>
+                <div class="updates-list">
```

Comment:
One more change here: Can we please replace `updates-list` (no longer needed) with `govuk-!-margin-top-3` to fix an issue with no spacing between the list of updates and the accordion header / button

Reply:
Ah, good spot. Have removed `updates-list` and add `govuk-!-margin-top-3`.